public/js/routing/router.js

```diff
@@ -72,12 +78,23 @@ export default class Router {
       this._updateContent();
       return;
     }
-    const location = this._window.document.location.href;
+    const location = this._document.location.href;
     const route = this.findRoute(location);
     this._takeOverAnchorLinks(this._container);
     route.onAttached();
   }
 
+  _dispatchNavigateEvent(url, route) {
+    const event = this._document.createEvent('CustomEvent');
```

Comment:
This is using the old syntax to create [CustomEvents](https://developer.mozilla.org/en-US/docs/Web/API/CustomEvent/CustomEvent).

The newer one is like this:
```
const event = new CustomEvent("cat", {
  detail: {
    hazcheeseburger: true
  }
});
obj.dispatchEvent(event);
`
[caniuse.com](http://caniuse.com/#search=CustomEvent) says the newer syntax doesn't work well in IE 11. Not sure how Babel handles this.